File: EventApp/src/app/shared/EventItem.ts
export class EventItem{
    Id!:number
    Type!:string
    Title!:string
    Short_title?:string
    Datetime_utc?:Date
    Visible_at?:Date|null
    Visible_until?:Date|null
    Venue!:Venue
    Performers: Performer[]=[]
}
export class Venue{
    Name?:string
    Address?:string
    City?:string
    State?:string
    Country?:string
}
export class Performer{
    Id?:number
    Name?:string|null
    Image?:string|null
}

File: APIGateway/EventService/Models/EventItem.cs
namespace EventService.Models
{
    public class EventItem
    {
        public int Id { get; set; }
        public string? Type { get; set; }
        public string? Title { get; set; }
        public string? Short_title { get; set; }
        public DateTime? Datetime_utc { get; set; }
        public DateTime? Visible_at { get; set; }
        public DateTime? Visible_until { get; set; }
        public Venue? Venue { get; set; }
        public List<Performer>? Performers { get; set; }

    }
}
